package-scripts.js

```diff
@@ -15,8 +15,8 @@ function test(testName, mochaParams) {
   if (process.env.CI) {
     // suppress coverage summaries in CI to reduce noise
     coverageCommand += ' --reporter=json';
-    if (!/^only-/.test(testName)) {
-      mochaParams += ' --forbid-only';
+    if (/^only-/.test(testName)) {
+      mochaParams += ' --no-forbid-only';
```

Comment:
[Question] 🤔 Is `--no-forbid-only` necessary? My interpretation of https://github.com/mochajs/mocha/issues/3030#issuecomment-333044369 was that the issue can be resolved with just the change to `forbid-only`'s default value.